src/entities/cart-item.ts

```diff
@@ -0,0 +1,7 @@
+import Product from "./product";
```

Comment:
In order to be consistent, all the files in the 'entities' folder should be named with PascalCase (CartItem.ts).